test/useEffectReducer.test.tsx

```diff
@@ -16,22 +16,18 @@ describe('useEffectReducer', () => {
   afterEach(cleanup);
 
   it('basic example', async () => {
-    interface User {
```

Comment:
The new type actually uncovered a typing issue in your tests in that you were passing strings as `User`